Xamarin.Forms.Platform.UAP/CarouselView/FormsViewToNativeUWP.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+    public static class FormsViewToNativeUWP
+    {
+        public static FrameworkElement ConvertFormsToNative(Xamarin.Forms.View view, Rectangle size)
+        {
+            //var vRenderer = RendererFactory.GetRenderer (view);
```

Comment:
Anything commented is intentional and there's a reason for that. The platform has some limitations. What is commented is the way it supposed to be :)